Onsets.py

```diff
@@ -0,0 +1,80 @@
+from numpy import *
+import datetime
+
+def sort_bombelkowe(tab):
+    for i in range(len(tab)):
+        x = len(tab) - 1
+        while x > i:
+            if tab[x] < tab[x - 1]:
+                z = tab[x]
+                tab[x] = tab[x - 1]
+                tab[x - 1] = z
+            x=x-1
```

Comment:
jak stosujesz spacje wokół operatorów to dobrze, ale wszędzie, proszę. Poczytaj o konwencji stawiania spacji w kodzie.